src/components/SearchBar.tsx

```diff
@@ -85,7 +85,7 @@ export interface SearchBarCssClasses extends AutocompleteResultCssClasses {
  * @remarks
  * The autocomplete loading state is passed in as an optional param.
  *
- * An onSubmit function is provided to allow an entity preview to be submitted.
+ * Default props for rendering corresponding DropdownItems are passed in.
```

Comment:
can you add more description about the props? and add description to the pr as well